debian/rules

```diff
@@ -97,7 +97,7 @@ DEPENDS += pam-challenge-response, \
 
 # Platform-specific dependencies
 DEPENDS.aws = nvme-cli,
-DEPENDS.azure = walinuxagent,
+DEPENDS.azure = walinuxagent, azure-cli,
```

Comment:
This change was made as part of https://delphix.atlassian.net/browse/DOSE-1219. Let's open a backport in jira and mark it as "fixed" in 6.0.16.0 as soon as you merge this PR to make sure there's a paper trail of this change having been backported.

Reply:
sounds good.. I'll wait to land this until we first land: https://github.com/delphix/delphix-platform/pull/382